pkg/decode/decode_protobuf.go

```diff
@@ -136,6 +137,15 @@ func RecordToMap(fr *flow.Record) config.GenericMap {
 	if fr.TimeFlowRtt != 0 {
 		out["TimeFlowRttNs"] = fr.TimeFlowRtt.Nanoseconds()
 	}
+
+	var ovsMetadatas []string
+	for _, md := range fr.OvsMonitorMD {
+		if md != "" {
+			ovsMetadatas = append(ovsMetadatas, md)
+		}
+	}
+	out["OvsMonitorMDs"] = ovsMetadatas
```

Comment:
@msherif1234 as discussed I would like to merge metadatas in a single field as we did for Interfaces and Directions

Reply:
done